faker/providers/address/zh_CN/__init__.py

```diff
@@ -22,10 +22,12 @@ class Provider(AddressProvider):
         "{{province}}{{city}}{{district}}{{street_address}} {{postcode}}",)
 
     provinces = (
-        "内蒙古自治区", "山西省", "河北省", "吉林省", "江苏省", "辽宁省", "黑龍江省",
+        "北京市", "上海市", "天津市" , "重庆市",
```

Comment:
can you remove the space before the 3rd comma?

Reply:
Sorry about that.Fixed now.